studio/schemas/documents/lesson.js

```diff
@@ -103,6 +103,49 @@ export default {
         layout: 'radio',
       },
     },
+    {
+      name: 'isCommunityResource',
+      title: 'Community Resource?',
+      type: 'boolean',
+    },
+    {
+      name: 'thumbnailUrl',
+      title: 'Thumbnail URL',
+      type: 'url',
+    },
+    {
+      name: 'iconUrl',
+      title: 'Icon URL',
+      type: 'url',
+    },
+    {
+      name: 'repoUrl',
+      title: 'Repo URL',
+      type: 'url',
+    },
+    {
+      name: 'codeUrl',
+      title: 'Code URL',
+      type: 'url',
+    },
```

Comment:
These values are still up for discussion: the source code only ever uses one or the other.

The `repoUrl` always ends up being a github URL built by a service. Seems unnecessary for this system to do the same, just past in whatever the URL is.

If we just had `codeUrl`, then it could point to github, codesandbox, jsbin, codepen, or wherever.

I guess the two values are handled slightly differently and/or potentially both displayed. So, something to think about how to best represent in the schema. https://github.com/eggheadio/egghead-next/blob/main/src/pages/lessons/%5Bslug%5D.tsx#L685-L714